tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -260,6 +264,10 @@ def _gen_kernel_library(
       extra_args: Extra arguments to pass to the generator tool.
       test_tags: The tags to pass to the generated test.
       test_size: The "size" argument to pass to the test.
+      jit_i64_indexed_for_large_tensors_types: The type to apply to the Kernel's 
```

Comment:
nit: The option is not very self-explanatory. I think a more detailed description could help people to understand this better.